test/expected/issue_533.out

```diff
@@ -0,0 +1,382 @@
+begin;
+    savepoint a;
+    create function public.uid()
+    returns uuid stable language sql as
+    $$
+      select 'adfc9433-b00e-4072-bbe5-5ae23a76e15d'::uuid
+    $$;
+    create function public.get_user_id(p_user_id uuid default public.uid())
+    returns uuid stable language sql as
+    $$
+        select p_user_id;
+    $$;
+    select jsonb_pretty(graphql.resolve($$
+    query IntrospectionQuery {
+        __schema {
+            queryType {
+                fields {
+                    name
+                    description
+                    type {
+                        kind
+                    }
+                    args {
+                        name
+                        defaultValue
+                        type {
+                            name
+                            kind
+                            ofType {
+                              name
+                              kind
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    } $$));
+                              jsonb_pretty                              
+------------------------------------------------------------------------
+ {                                                                     +
+     "data": {                                                         +
+         "__schema": {                                                 +
+             "queryType": {                                            +
+                 "fields": [                                           +
+                     {                                                 +
+                         "args": [                                     +
+                             {                                         +
+                                 "name": "pUserId",                    +
+                                 "type": {                             +
+                                     "kind": "SCALAR",                 +
+                                     "name": "UUID",                   +
+                                     "ofType": null                    +
+                                 },                                    +
+                                 "defaultValue": "uid()"               +
```

Comment:
I don't think we want to show the SQL logic of the default value here

Would it make sense to make the field nullable and not display the default value. I think we've used that approach in the past?

Reply:
If I understood correctly, your concern is only about the default value being a SQL expression, because the field is already nullable, right? In that case, we can replace `uid()` here with an empty string or something like `<omittted>` to indicate the complex nature of the default expression. We should probably also update the docs to explain why default value is an empty string or `<omitted>`.

Reply:
> concern ... default value being a SQL expression

yeah exactly. I think it should be nullable (already done) and the `defaultValue` should be NULL.

Thats how Postgraphile handles it (screenshot), so seems like a safe bet

<img width="239" alt="Screenshot 2024-06-26 at 11 09 04 AM" src="https://github.com/supabase/pg_graphql/assets/12958657/ffac215f-0ede-4cd3-810d-440a82bc7db0">

Reply:
Done.